lib/plugin/instance.rb

```diff
@@ -117,6 +117,20 @@ def add_admin_route(label, location, opts = {})
     }
   end
 
+  def full_admin_route
+    route = self.admin_route
+    return unless route
+
+    ret = route.slice(:location, :label)
+    if route[:use_new_show_route]
+      ret[:full_location] = "adminPlugins.show"
+      ret[:use_new_show_route] = true
+    else
+      ret[:full_location] = "adminPlugins.#{ret[:location]}"
+    end
+    ret
```

Comment:
I think right now the `use_new_show_route` key is entirely omitted if the value is `false`. I think it's better to still include it and set the value to `false`. WDYT?

```suggestion
    route.slice(:location, :label, :use_new_show_route).tap do |admin_route|
      path = admin_route[:use_new_show_route] ? "show" : admin_route[:location]

      admin_route[:full_location] = "admin_plugins.#{path}"
    end
```

Reply:
Looks nice, will use this :+1: